lib/collection/response.js

```diff
@@ -11,6 +11,14 @@ var util = require('../util'),
     Cookie = require('./cookie').Cookie,
     PropertyList = require('./property-list').PropertyList,
 
+    normaliseStream = function (stream) {
+        if (stream && stream.type === 'Buffer' && _.isArray(stream.data)) {
+            // @todo Add tests for Browser environments, where ArrayBuffer is returned instead of Buffer
+            return typeof Buffer === 'function' ? new Buffer(stream.data) : new Uint8Array(stream.data).buffer;
```

Comment:
Also, can you double check and confirm Uint8Array().buffer actually works in browsers? Also move the 'function' string to a variable on top.